recipe/meta.yaml

```diff
@@ -1,63 +1,126 @@
-{% set name = "pennylane-lightning" %}
-{% set version = "0.24.0" %}
+{% set name = "pennylane" %}
+{% set version = "0.29.0" %}
 
 
 package:
   name: {{ name|lower }}
```

Comment:
So PL is an empty package that depends on core and lightning?

Reply:
And also ref https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#outputs-section

Reply:
Ok in that case you should probably use an empty meta package and have the package be one of the outputs.

One example is matplotlib, where the top package is just called `-suite` but is not actually installable: https://github.com/conda-forge/matplotlib-feedstock/blob/main/recipe/meta.yaml

(See also the note here https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#outputs-section)